pkg/receive/otlptranslator/context.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+// Provenance-includes-location: https://github.com/prometheus/prometheus/blob/318d6bc4bfe2d81af88697940740a378b503335a/storage/remote/otlptranslator/prometheusremotewrite/context.go#L18
```

Comment:
Just for my understanding, where are these provenance directives from? What does it mean? That the code is copied from Prometheus repo?

Reply:
I copy the code from prometheus and prometheus copied from OTel 😅